spec/tasks/translation_spec.rb

```diff
@@ -1,16 +1,62 @@
-require 'spec_helper'
-require_relative '../../spec/support/tasks'  
+require "spec_helper"
+require_relative "../../spec/support/tasks"
 
-describe 'rake tasks' do
+describe "rake tasks" do
   before do
     fake_rails = double()
-    fake_rails.stub(:root) { Pathname.new('spec') }
+    fake_rails.stub(:root) { Pathname.new("spec") }
     stub_const("Rails", fake_rails)
   end
 
+  describe "translation:import", type: :task do
+    let(:task) { Rake::Task["translation:import"] }
+    let(:csv_path) { "path/to/import/fr.csv" }
+    let!(:importer_instance) { stub_importer }
```

Comment:
As a general rule, we should avoid using `let` when the variable is only used in one test. This one, for example, can be moved into a `importer_instance = stub_importer` on line 23, as it is only used in one text inside this `describe` block.

Same can be said for the one around line 53, and then both `cleaner_instance` declarations in the last commit.

Reply:
Agreed. However, as the `let` is actually setting up stubs, rather than just assigning a variable then we'd need to run `stub_importer` in both tests anyway. We're sort of misusing `let` here as the stubs would ideally live in a `before` block, but this isn't very DRY. I don't think its too bad though as `let!` gives a good indication to the developer that the logic needs to be executed at startup.

Reply:
Merging but will happily raise another PR for further changes regarding the spec

Reply:
I'd missed that nuance - agreed to leave as-is. Thanks!